packages/hoppscotch-backend/src/user/user.service.ts

```diff
@@ -41,12 +41,8 @@ export class UserService {
 
     return {
       ...dbUser,
-      currentRESTSession: dbCurrentRESTSession
-        ? JSON.stringify(dbCurrentRESTSession)
-        : null,
-      currentGQLSession: dbCurrentGQLSession
-        ? JSON.stringify(dbCurrentGQLSession)
-        : null,
+      currentRESTSession: JSON.stringify(dbCurrentRESTSession) ?? null,
+      currentGQLSession:  JSON.stringify(dbCurrentGQLSession) ?? null,
```

Comment:
`dbCurrentRESTSession` can be null.
`JSON.stringify(null)` will return `"null"` as string data-type, which breaks the contract.

Same goes for `dbCurrentGQLSession`.